nats-base-client/types.ts

```diff
@@ -136,18 +137,28 @@ export interface Base {
 }
 
 export interface ServerInfo {
+  server_id: string;
+  server_name: string;
+  version: string;
+  proto: number;
+  git_commit?: string;
+  go: string;
+  host: string;
+  port: number;
+  headers?: boolean;
+  auth_required?: boolean;
   tls_required?: boolean;
   tls_verify?: boolean;
-  connect_urls?: string[];
+  tls_available?: boolean;
   max_payload: number;
-  client_id: number;
-  headers?: boolean;
-  proto: number;
-  server_id: string;
-  version: string;
-  echo?: boolean;
+  jetstream?: boolean;
+  client_id?: number;
+  client_ip?: string;
   nonce?: string;
-  nkey?: string;
+  cluster?: string;
+  connect_urls?: string[];
+  ws_connect_urls?: string[];
```

Comment:
This is used only between servers. The client will get it from `connect_urls`. For WS clients, it will be WS urls. In other words, the server always send `connect_urls` but adapt the content based on the type of client.

Reply:
awesome thanks.